webapp/src/main/java/com/box/l10n/mojito/service/branch/notification/github/BranchNotificationMessageSenderGithub.java

```diff
@@ -86,7 +86,7 @@ public void sendTranslatedMessage(String branchName, String username, String mes
       githubClient.addCommentToPR(
           branchDetails.getRepository(),
           branchDetails.getPrNumber(),
-          branchNotificationMessageBuilderGithub.getTranslatedMessage());
+          branchNotificationMessageBuilderGithub.getTranslatedMessage(branchName));
```

Comment:
should we pass the branch detail here, vs re-splitting?

Reply:
yes, will change, thanks